eZ/Bundle/EzPublishCoreBundle/Resources/translations/repository_exceptions.pt.xlf

```diff
@@ -1,113 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like the exceptions were _moved_ to messages. It shouldn't have happened.